server/controller/userController.js

```diff
@@ -0,0 +1,116 @@
+import jwt from 'jsonwebtoken';
+import pg from 'pg';
+
+import config from './../config/config';
+import user from './../seed/user';
+
+const { connect } = user;
+
+const client = new pg.Client(config);
+
+require('dotenv').config();
+
+connect();
+
+/**
+ * it is a class that control all event method
+ */
+class UserController {
+  /**
+ * it create a new user
+ * @param {string} req
+ * @param {string} res
+ * @return {object} an object
+ */
+  static async signUp(req, res) {
+    try {
+      const {
+        firstName, lastName, phoneNumber, email, password
+      } = req.body;
+
+      const checkEmail = `
+            SELECT * 
+            FROM users
+            WHERE email = '${email}'
+      `;
+      const query = `
+            INSERT INTO users (
+                first_name,
+                last_name,
+                phone_number,
+                email,
+                password
+            ) 
+            VALUES (
+                '${firstName}',
+                '${lastName}',
+                '${phoneNumber}',
+                '${email}',
+                crypt('${password}', gen_salt('${process.env.KEY}', 5))
+            ) RETURNING *;    
+        `;
+
+      client.connect();
+      const foundEmail = await client.query(checkEmail);
+
+      if (foundEmail.rows[0]) {
+        return res.status(409).json({
+          status: 'error',
+          message: 'email is already existing'
+        });
+      }
+
+      const result = await client.query(query);
+      res.status(201).json({
+        status: 'success',
+        data: {
+          newUser: result.rows[0]
+        },
+        message: 'user created successfully'
+      });
+    } catch (error) { res.status(500).send(error); }
+  }
+
+  /**
+ * it create a new user
+ * @param {string} req
+ * @param {string} res
+ * @return {object} an object
+ */
+  static async login(req, res) {
+    try {
+      const {
+        email, password
+      } = req.body;
+
+      const checkEmail = `
```

Comment:
Why is this variable called checkEmail when it checks both email and crypted password?